src/tests/Microsoft.PowerFx.Interpreter.Tests/ConfigTests.cs

```diff
@@ -339,45 +338,49 @@ public void RecalcEngine_Symbol_CultureInfo3()
 
             check = engine.Check(datetimeExpression).GetEvaluator();
 
-            Assert.Equal("10/06/2022 14:19", (check.Eval() as StringValue).Value);
+            Assert.Equal("10/6/2022 2:19 PM", (check.Eval() as StringValue).Value);
```

Comment:
>10/6/2022 2:19 PM" [](http://example.com/codeflow?start=26&length=18)

Why did this test result change?

Reply:
Due to change at line 915/920, this is expected